src/Pmi/Entities/Participant.php

```diff
@@ -338,7 +340,7 @@ private function getSiteSuffix($site)
 
     private function getActivityStatus($participant)
     {
-        if ($participant->withdrawalStatus === 'NO_USE') {
+        if (in_array($participant->withdrawalStatus, ['NO_USE', 'EARLY_OUT'], true)) {
```

Comment:
I wasn't able to centralize the logic for this one because `getActivityStatus()` is taking a `StdClass` value rather than a `Participant`.

Reply:
Makes sense. Maybe put these two values in a static class array so that it can be used in both places?

Reply:
Done in 30ca5ba